### PR TITLE
Feature/ldp 697

### PR DESCRIPTION
**Pull request by maximilianmikus**

### CODE REVIEW

File: nuxt.config.js [hunk 1]
```diff
@@ -35,7 +37,9 @@ export default {
   ],
 
   'nuxtjs-drupal-ce': {
-    baseURL: 'http://example.d9playground.localdev.space'
+    baseURL: DRUPAL_BASE_URL,
```
Comment (fago) [inline nuxt.config.js]: the module should automatically apply DRUPAL_BASE_URL even without your code, it already has that logic in place. You can you try just setting the baseurl?

Comment (maximilianmikus) [reply nuxt.config.js]: I tried, but it was not working so i fixed it here - it gets explicitly overwritten in line186 plugin.js `options.baseURL = runtimeConfig.baseURL || options.baseURL` - runtimeConfig.baseURL takes precedence

Comment (fago) [reply nuxt.config.js]: ok, so let's open a bug report at the module then and fix it separately?

**Review by fago — APPROVED** [review]

**Comment by fago** [comment]
this looks all good but changes the packages to a branch. we must fix this before merge.

**Review by fago — APPROVED** [review]

**Pull request merged**